### PR TITLE
sig-net code jammmm

### PR DESCRIPTION
**Pull request by jayunit100**
per https://groups.google.com/g/kubernetes-sig-network/c/sJtBHe0Tx5o, lets broaden the gateway codejam and get rid of the kpng hangout

### CODE REVIEW

**Comment by jayunit100** [comment]
/tide/merge-method-rebase

File: sig-network/README.md [hunk 1]
```diff
@@ -26,14 +25,13 @@ The [charter](charter.md) defines the scope and governance of the Network Specia
...
 * SIG Network Meeting: [Thursdays at 09:00 PT (Pacific Time)](https://zoom.us/j/361123509) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=09:00&tz=PT%20%28Pacific%20Time%29).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1_w77-zG_Xj0zYvEMfQZTQ-wPP4kXkpGD8smVtW_qqWM/edit).
   * [Meeting recordings](https://www.youtube.com/watch?v=phCA5-vWkVM&list=PL69nYSiGNLP2E8vmnqo5MwPOY25sDWIxb).
 * SIG Network Multi-Network Meeting: [Wednesdays at 08:00 PT (Pacific Time)](https://zoom.us/j/95680858961?pwd=M1c2TTdMZHpMUUtIYXRpbjRobkNJZz09) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:00&tz=PT%20%28Pacific%20Time%29).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1pe_0aOsI35BEsQJ-FhFH9Z_pWQcU2uqwAnOx2NIx6OY/edit).
   * [Meeting recordings](https://www.youtube.com/watch?v=phCA5-vWkVM&list=PL69nYSiGNLP2E8vmnqo5MwPOY25sDWIxb).
+* Sig-Network and Gateway API Code Jam: [Fridays at 08:30 PT (Pacific Time)](https://zoom.us/j/96900767253) (Last Friday of the Month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:30&tz=PT%20%28Pacific%20Time%29).
```
Comment (shaneutt) [inline sig-network/README.md:34]: I honestly think you can drop the `Gateway API` part from this, I think we're really trying to make this just a generic SIG Network sync.

```suggestion
* Sig-Network Jam: [Fridays at 08:30 PT (Pacific Time)](https://zoom.us/j/96900767253) (Last Friday of the Month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:30&tz=PT%20%28Pacific%20Time%29).
```

**Comment by jayunit100** [comment]
okidoki, just pusheed

**Comment by jayunit100** [comment]
btw @shaneutt you have to edit `sigs.yaml` and then run `make` to update the file, cant directly edit the readme :) but yup, itegrated your change

File: sig-network/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ The [charter](charter.md) defines the scope and governance of the Network Specia
 * SIG Network Multi-Network Meeting: [Wednesdays at 08:00 PT (Pacific Time)](https://zoom.us/j/95680858961?pwd=M1c2TTdMZHpMUUtIYXRpbjRobkNJZz09) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:00&tz=PT%20%28Pacific%20Time%29).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1pe_0aOsI35BEsQJ-FhFH9Z_pWQcU2uqwAnOx2NIx6OY/edit).
   * [Meeting recordings](https://www.youtube.com/watch?v=phCA5-vWkVM&list=PL69nYSiGNLP2E8vmnqo5MwPOY25sDWIxb).
-* Sig-Network and Gateway API Code Jam: [Fridays at 08:30 PT (Pacific Time)](https://zoom.us/j/96900767253) (Last Friday of the Month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:30&tz=PT%20%28Pacific%20Time%29).
+* Sig-Network Jam: [Fridays at 08:30 PT (Pacific Time)](https://zoom.us/j/96900767253) (Last Friday of the Month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:30&tz=PT%20%28Pacific%20Time%29).
```
Comment (shaneutt) [inline sig-network/README.md]:
```suggestion
* Sig-Network Code Jam: [Fridays at 08:30 PT (Pacific Time)](https://zoom.us/j/96900767253) (Last Friday of the Month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:30&tz=PT%20%28Pacific%20Time%29).
```

Comment (shaneutt) [reply sig-network/README.md]: sorry, meant to keep the "Code Jam" part I think that's an important part of its current identity

Comment (jayunit100) [reply sig-network/README.md]: ya definetly

**Comment by jayunit100** [comment]
I think it's all set now right @shaneutt ?

**Review by shaneutt — APPROVED** [review]
/lgtm

:vulcan_salute: 

cc @mrbobbytables

**Review by mrbobbytables — COMMENTED** [review]
/approve

**Pull request merged**